06_coefficient_analysis.ipynb

```diff
@@ -0,0 +1,848 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Analysis of model coefficients"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
+    "\n",
+    "import pancancer_evaluation.config as cfg\n",
+    "import pancancer_evaluation.utilities.data_utilities as du\n",
+    "import pancancer_evaluation.utilities.analysis_utilities as au"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "results_dir = '/home/jake/data/pancancer_vogelstein_09_25'\n",
```

Comment:
Not a good reason - I just did this because I'm running some other experiments, and didn't want them to overwrite my previous results. I'll get rid of this.